zaycevnet/content/contents/code/zaycev.js

```diff
@@ -0,0 +1,136 @@
+/* zaycevnet.info resolver for Tomahawk.
+ *
+ * Written in 2015 by Anton Romanov
+ * Licensed under the Eiffel Forum License 2.
+ *
+ */
+
+var ZaycevResolver = Tomahawk.extend( Tomahawk.Resolver.Promise, {
+    apiVersion: 0.9,
+
+    settings: {
+        cacheTime: 300,
+        name: 'zaycevnet',
+        weight: 76,
+        icon: '../images/icon.png',
+        timeout: 8
+    },
+
+    rc4: function (key, str) {
```

Comment:
Have you implemented this function yourself? Otherwise you need to provide its origin / copyright notice...